products/libllbuild/C-API.cpp

```diff
@@ -25,3 +25,7 @@ const char* llb_get_full_version_string(void) {
 
   return versionString.c_str();
 }
+
+int llb_get_c_api_version(void) {
```

Comment:
Should this be llb_get_api_version? I guess its obvious it is the C API version?